todo_app/spec/models/task_spec.rb

```diff
@@ -26,4 +26,16 @@
       it { is_expected.to_not be_valid }
     end
   end
+
+   describe '#scope sort_tasks' do
+   let!(:task1) { create(:task, due_date: Faker::Time.backward) }
+   let!(:task2) { create(:task, due_date: Faker::Time.forward) }
```

Comment:
taskが2つくらいならこのままでも大丈夫かなという気はしなくもないのですが、できればtaskの内容が伝わるような名前を付けていただけると後々のメンテ性も上がって嬉しいです！

Reply:
承知致しました、titleつけてみます！

Reply:
こちらの指摘は対応漏れですかね…？
対応済みかどうか分かるようにコメント残してもらえるととてもありがたいですー
私はたまに以下のようにコメント一緒に修正commitを貼り付けたりします！
例：変数の名前を変更しました https://github.com/Fablic/training/pull/993/commits/fb12cd1dfb36116397af62c52015f914dbd7c64c

Reply:
対応漏れです。タイトル追加しました。
https://github.com/Fablic/training/pull/993/commits/b8457521729acf44d6ea0b6b1c11856dccfb9836

Reply:
ああー、すみません、ちょっと言葉足らずでしたmm
`task1` のような名前の付け方だとマジックナンバーとなってしまい、その変数が持つインスタンスがどのようなものなのか分かりづらくなってしまうんじゃないかといった意味でした！
あと、修正されているのがsystem specのファイル（spec/system/tasks_spec.rb）なのですが、modelの方（spec/models/task_spec.rb）も修正していただけると嬉しいですー！

```suggestion
   let!(:old_task) { create(:task, due_date: Faker::Time.backward) }
   let!(:new_task) { create(:task, due_date: Faker::Time.forward) }
```

Reply:
追加しました。
https://github.com/Fablic/training/pull/993/commits/c417ca820dca1e855a59a1fc910ec417924a8862
https://github.com/Fablic/training/pull/993/commits/429ef4fc418c0eb43006bea65dcb34cb4b1197f7